### ISSUE TITLE
Consider an e-commerce Template with Medusa.js

### ISSUE THREAD

**Issue by Devonte202** [amiet1/EnerFinal#7]
## What is Medusa.js

[Medusa.js](https://medusajs.com/) is essentially an e-commerce backend that comes with all of the APIs necessary for a storefront built in. Meaning it essentially requires no work to be done on the backend.

#### Things to consider

- This will mean abandoning your current backend to learn a new backend. 

- The upside here is that you won't have to worry about building it anymore, you'll get to just focus on your frontend.

- You still get to talk about the process of building your current backend during interviews and explaining why you pivoted to Medusa in the end. This shows experience, and a willingness to pivot when a solution doesn't work out. 

- Its also the perfect answer to the question: > "Tell me about a challenging experience you had during a project". You then get to talk about how you first hand discovered how difficult it is to build an e-commerce shop site from scratch

----------------------
## Try using Medusa.js

You can actually spin up a default Medusa site the same way you can spin up a react site. Just run this command

NOTE: Make sure you `cd` out of Ener before doing this

```
npx create-medusa-app@latest
```

You''ll then have to answer a few prompts in the terminal. For the postgres ones just press enter and don't type anything

Once this is done (It might take a little long, because it has to install a bunch of stuff) It'll then open up your localhost and ask you to create an admin user with an email and a password.

<img width="640" alt="Screenshot 2024-06-08 at 11 38 16 AM" src="https://github.com/amiet1/EnerFinal/assets/14931357/e3a4d264-acc3-4dc3-93c8-045c1399802f">

This is the account you'll be using to manage your Medusa backend (Adding products, managing orders, etc)

There's a mini tutorial you'll get to go through before you get taken to the Frontend.
-----------------------

If the suspense is killing you and you just wanna see what a Medusa site looks like, check this one out:

https://next.medusajs.com/us

You can scroll down from here and see the products, and test out the checkout process.

## Why Medusa.js

In short E-commerce sites are a nightmare to build, so using a template like this with an entire team of engineers maintaining it, is a pretty safe option, especially since you want to actually use this for real customers. 

Using this allows you to focus on mostly the frontend without having to worry about backend. It'll also speed up the process of building Ener, dramatically. Instead of weeks we'll be looking at days.
